101-webapp-basic-windows/README.md

```diff
@@ -1,4 +1,5 @@
-# Deploy a app service plan and a basic Windows web app
+# This template allows you to deploy an App Service Plan and a Windows web app.
```

Comment:
Change this to title "voice", e.g.

Windows App Service Plan and Basic Web App